### PR TITLE
AR-871: Added --update --cleanup-regions to prod script

### PR DESCRIPTION
**Pull request by tuj**

### CODE REVIEW

**Review by sinejespersen — APPROVED** [review]

File: restart-prod.sh [hunk 1]
```diff
@@ -20,10 +20,10 @@ docker-compose --env-file .env.docker.local -f docker-compose.server.yml exec --
...
-docker-compose --env-file .env.docker.local -f docker-compose.server.yml exec --user deploy api bin/console app:screen-layouts:load https://raw.githubusercontent.com/os2display/display-templates/develop/src/screen-layouts/two-boxes.json
-docker-compose --env-file .env.docker.local -f docker-compose.server.yml exec --user deploy api bin/console app:screen-layouts:load https://raw.githubusercontent.com/os2display/display-templates/develop/src/screen-layouts/two-boxes-vertical.json
-docker-compose --env-file .env.docker.local -f docker-compose.server.yml exec --user deploy api bin/console app:screen-layouts:load https://raw.githubusercontent.com/os2display/display-templates/develop/src/screen-layouts/six-areas.json
+docker-compose --env-file .env.docker.local -f docker-compose.server.yml exec --user deploy api bin/console app:screen-layouts:load --update --cleanup-regions https://raw.githubusercontent.com/os2display/display-templates/develop/src/screen-layouts/full-screen.json
```
Comment (turegjorup) [inline restart-prod.sh:23]: Shouldn't these link to `main`?

**Review by turegjorup — CHANGES_REQUESTED** [review]
Why only change the `restart-prod.sh` script? Is this not needed for `dev/stg`?

File: restart-prod.sh (hunk 1 above)
Comment (tuj) [reply restart-prod.sh]: Fixed

**Comment by tuj** [comment]
The restart_develop script has been updated in another PR

**Review by turegjorup — APPROVED** [review]

**Pull request merged**